rocAL/rocAL_pybind/setup.py

```diff
@@ -28,7 +28,7 @@ def run(self):
     def build_extension(self, ext):
         extdir = os.path.abspath(os.path.dirname(self.get_ext_fullpath(ext.name)))
         cmake_args = ['-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=' + extdir,
-                      '-DPYTHON_EXECUTABLE=' + sys.executable, '-DGPU_SUPPORT=ON', '-DBACKEND=OPENCL']
+                      '-DPYTHON_EXECUTABLE=' + sys.executable, '-DGPU_SUPPORT=ON', '-DBACKEND=HIP'] #Needs to be changed back to OCL for OCL backend
```

Comment:
add option in CMake to take it from command line

Reply:
We are now able to pass the user given cmd line args in run.sh & use the same in setup.py file to build either the OCL / HIP backends.
The HIP is added as the default backend while running the run.sh file